client/src/Shortcut.re

```diff
@@ -53,10 +53,14 @@ module Store = {
     };
   };
 
+  /* single source of truth for this key */
+  let refmtShortcutKey = "ctrl+shift+i";
+
   let subscribe: subscribeFun =
     (~global=false, key, handler) => {
-      if ((m^)->ShortcutMap.has(key)) {
-        warn("Overiding " ++ key ++ " shortcut");
+      /* Don't warn when overriding refmt shorcut since re-registering on each lang update */
```

Comment:
I'm not sure why is this needed? I don't see you registering this anywhere

Reply:
You're right, this is no longer needed. Forgot to remove it.